futurex_openedx_extensions/helpers/tenants.py

```diff
@@ -101,6 +101,24 @@ def get_all_tenants_info() -> Dict[str, str | dict | List[int]]:
     }
 
 
+def get_fx_dashboard_url(request: Any, user_accessible_tenant_ids: List[int]) -> str:
+    """
+    Get Fx Dashboard URL
+
+    :param request: Current django request.
+    :type request: Request
+    :param user_accessible_tenant_ids: Tenant ids accessible to User.
+    :type user_accessible_tenant_ids: List[int]
+    :return: Fx Dashboard URL
+    :rtype: str
+    """
+    tenant_info = get_all_tenants_info()
```

Comment:
Please use a more efficient method. As far as I know this method isn't efficient and could contradict with the https://github.com/eduNEXT/eox-tenant logic which we rely on.

Please use a more standarized method to `get_current_tenant_config`·

@shadinaif what's the best way to do that because I couldn't find this method in the `eox-tenant` package?

Reply:
This method is cached and it uses the same logic here. But there is still a performance issue in another place which is `get_accessible_tenant_ids`. I'll discuss that with @tehreem-sadat today